index.js

```diff
@@ -42,4 +40,13 @@ function generateRandomFileName(fileNumber){
   return 'placeholder_' + commanderConfig.size + '_' + randomString({length: 4}) + fileNumber + randomString({length: 4}) + '.jpg'
 }
 
+function httpRequest(url, Response){
+  'use strict';
+  if (url.substring(0, 7) === 'http://') {
```

Comment:
```suggestion
  if (isHttp(url)) {
```
You could extract the condition in a function to avoid mixing abstraction levels.